egs/aspire/s5/run.sh

```diff
@@ -159,30 +165,55 @@ steps/train_sat.sh  --cmd "$train_cmd" \
+ --ivector-scale 0.75 --affix v6 --tune-hyper true test_aspire data/lang exp/nnet2_multicondition/nnet_ms_a
 # 72.3 on leaderboard
 
 # discriminative training. Helped on dev, but not on dev_test
 local/multi_condition/run_nnet2_ms_disc.sh
- local/multi_condition/prep_test_aspire.sh --stage 1 --decode-num-jobs 200 \
-   --sub-speaker-frames 6000 --window 10 --overlap 5 --max-count 75 --pass2-decode-opts "--min-active 1000" \
-   --ivector-scale 0.75 --affix v6 --tune-hyper true dev_aspire data/lang exp/nnet2_multicondition/nnet_ms_a_smbr_0.00015_nj12
- #%WER 29.1 | 2120 27208 | 77.6 15.4 7.0 6.7 29.1 77.1 | -1.357 | exp/nnet2_multicondition/nnet_ms_c_prior_adjusted_smbr_0.00015_nj12/decode_dev_aspire_whole_uniformsegmented_win10_over5_v6_iterepoch2_pp_fg/score_16/penalty_1.0/ctm.filt.filt.sys
-
- local/multi_condition/prep_test_aspire.sh --stage 1 --decode-num-jobs 200 \
-   --sub-speaker-frames 6000 --window 10 --overlap 5 --max-count 75 --pass2-decode-opts "--min-active 1000" \
-   --ivector-scale 0.75 --affix v6 --tune-hyper true test_aspire data/lang exp/nnet2_multicondition/nnet_ms_a_smbr_0.00015_nj12
- # around 71.5, as models changed after server closed
-
+local/multi_condition/prep_test_aspire.sh --stage 1 --decode-num-jobs 200 \
+ --sub-speaker-frames 6000 --window 10 --overlap 5 --max-count 75 --pass2-decode-opts "--min-active 1000" \
+ --ivector-scale 0.75 --affix v6 --tune-hyper true dev_aspire data/lang exp/nnet2_multicondition/nnet_ms_a_smbr_0.00015_nj12
+#%WER 29.1 | 2120 27208 | 77.6 15.4 7.0 6.7 29.1 77.1 | -1.357 | exp/nnet2_multicondition/nnet_ms_c_prior_adjusted_smbr_0.00015_nj12/decode_dev_aspire_whole_uniformsegmented_win10_over5_v6_iterepoch2_pp_fg/score_16/penalty_1.0/ctm.filt.filt.sys
+
+local/multi_condition/prep_test_aspire.sh --stage 1 --decode-num-jobs 200 \
+ --sub-speaker-frames 6000 --window 10 --overlap 5 --max-count 75 --pass2-decode-opts "--min-active 1000" \
+ --ivector-scale 0.75 --affix v6 --tune-hyper true test_aspire data/lang exp/nnet2_multicondition/nnet_ms_a_smbr_0.00015_nj12
+# around 71.5, as models changed after server closed
+
+local/chain/run_tdnn_lstm.sh
+# %WER 22.9 | 2083 25838 | 81.6 12.0 6.4 4.6 22.9 70.8 | -0.469 | exp/chain/tdnn_lstm_1a/decode_dev_aspire_uniformsegmented_win10_over5_v8_iterfinal_pp_fg/score_8/penalty_0.0/ctm.filt.filt.sys
+# %WER 24.0 | 2083 25822 | 79.9 11.9 8.1 3.9 24.0 71.7 | -0.450 | exp/chain/tdnn_lstm_1a_online/decode_dev_aspire_uniformsegmented_win10_over5_v8_iterfinal_pp_fg/score_10/penalty_0.0/ctm.filt.filt.sys
+
+# Train speech activity detection system using TDNN+Stats
+local/run_asr_segmentation.sh
+
+sad_nnet_dir=exp/segmentation_1a/tdnn_stats_asr_sad_1a
+chain_dir=exp/chain/tdnn_lstm_1a
+
+# %WER 23.2 | 2083 25819 | 81.9 11.9 6.2 5.1 23.2 71.8 | -0.641 | exp/chain/tdnn_lstm_1a/decode_dev_aspire_asr_sad_1a_iterfinal_pp_fg/score_8/penalty_0.75/ctm.filt.filt.sys
```

Comment:
Just an FYI : 
Oracle segmentation is available for dev data. I never cared to use it in the original recipe as the final test scenario didn't provide segmented recordings. However given that this LVCSR task is also being used for sad_segmentation testing I would recommend tracking the WER even with oracle segmentation.

Reply:
Did you put this oracle segmentation result or the corresponding script somewhere in the PR?

Reply:
I added this in the results, but there is no corresponding script.